docs/content/developers/release-management.md

```diff
@@ -179,3 +179,14 @@ This is done automatically by the release build on a dedicated Windows test runn
 
 !!!tip "We shouldn’t use this high-security keyfob approach to signing on the next go-around with the certs."
     It’s way too difficult to manage, and the Safenet software is atrocious.
+
+## APT and YUM/RPM Package Management
+
+The Linux `apt` and `yum/rpm` packages are built and pushed by the `nfpms` and `furies` sections of the [.goreleaser.yml](https://github.com/ddev/ddev/blob/master/.goreleaser.yml).
+
+* The actual packages are served by [gemfury.com](https://gemfury.com/).
+* The name of the organization in GemFury is [`drud`](`https://manage.fury.io/dashboard/drud`).
+* `rfay`, `mattstein`, and `gilbertsoft` are authorized as owners on this dashboard.
```

Comment:
Make these meaningfully identify human beings and qualify the handles in case this ends up being important:

```suggestion
* [Randy Fay](https://github.com/rfay), [Matt Stein](https://github.com/mattstein), and [Gilbertsoft](https://github.com/gilbertsoft) are authorized as owners on this dashboard.
```

Reply:
```suggestion
* [Randy Fay](https://github.com/rfay), [Matt Stein](https://github.com/mattstein), and [Simon Gillis](https://github.com/gilbertsoft) are authorized as owners on this dashboard.
```

Reply:
@rfay Simon didn’t put that name on the GitHub profile and the URL there 404s so I was trying to be respectful and not volunteer information. Obviously this change seems like a good one, provided we’re all comfortable with it!

Reply:
I'll ask him.